pkg/engine/runtime/kubernetes/kubernetes_runtime.go

```diff
@@ -225,17 +233,14 @@ func (k *KubernetesRuntime) Import(ctx context.Context, request *runtime.ImportR
 		if err != nil {
 			return &runtime.ImportResponse{Status: v1.NewErrorStatusWithCode(v1.IllegalManifest, err)}
 		}
-	} else {
+	} else if convertor.Service == ur.GetKind() {
 		// normalize resources
-		if convertor.Service == ur.GetKind() {
-			if err := normalizeService(ur); err != nil {
-				return &runtime.ImportResponse{
-					Resource: nil,
-					Status:   v1.NewErrorStatusWithCode(v1.IllegalManifest, err),
-				}
+		if err := normalizeService(ur); err != nil {
+			return &runtime.ImportResponse{
+				Resource: nil,
+				Status:   v1.NewErrorStatusWithCode(v1.IllegalManifest, err),
 			}
 		}
-		normalizeServerSideFields(ur)
```

Comment:
Why did you delete this line? `normalizeServerSideFields` and `normalizeService` are not the same.

Reply:
Currently, only the `Import` method handles `normalizeServerSideFields` for resource objects obtained from the cluster. This PR adds the process of `normalizeServerSideFields` in both `Read` and `Apply`, hence the `Import` method obtaining cluster resources via `Read` should no longer require additional processing.